spring-cloud-config-monitor/src/main/java/org/springframework/cloud/config/monitor/EnvironmentMonitorAutoConfiguration.java

```diff
@@ -31,7 +31,7 @@
  */
 @Configuration
 @ConditionalOnWebApplication
-@Import(FileMonitorConfiguration.class)
+@Import({FileMonitorConfiguration.class, PropertyPathNotificationExtractorConfiguration.class})
```

Comment:
Can you try with a `protected static class` in this same class rather than another class with an import?